### PR TITLE
Add eslint, and travis

### PR DESCRIPTION
**Pull request by nullpixel**
Added

* eslint config
* travis config

Updated readme to look better w/ travis badge

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,27 +1,282 @@
 module.exports = {
     "env": {
         "browser": true,
-        "es6": true,
-        "node": true
```
Comment (Mstrodl) [inline .eslintrc.js:5]: Why was this removed?

File: src/bot/commands/help.js [hunk 2]
```diff
@@ -15,12 +15,7 @@ function generateAllHelp(commands) {
     return helpText
 }
 
-function generateHelp(command) {
-    return `    ${command.trigger.join(" or ")}: ${command.description}`
-}
-
 function help(data) {
-    console.log("asdf")
```
Comment (Mstrodl) [inline src/bot/commands/help.js:23]: 👍

**Review by Mstrodl — CHANGES_REQUESTED** [review]
Pls explain kthx

File: .eslintrc.js (hunk 1 above)
Comment (nullpixel) [reply .eslintrc.js:5]: Added again 👍

**Review by Mstrodl — APPROVED** [review]

**Comment by Mstrodl** [comment]
Looks good
